bsp/boards/scum/leds.c

```diff
@@ -0,0 +1,124 @@
+/**
+\brief SCuM-specific definition of the "leds" bsp module.
+
+\author Tengfei Chang <tengfei.chang@inria.fr>, August 2016.
+*/
+
+#include "memory_map.h"
+#include "stdint.h"
+#include "leds.h"
+
+//=========================== defines =========================================
+
+//=========================== variables =======================================
+
+//=========================== prototypes ======================================
+
+//=========================== public ==========================================
+
+void leds_init() {
+    GPIO_REG__OUTPUT    &= ~0x0F;    // GPIO_REG__OUTPUT = 0bxxxx0000, all LEDs off
+}
+
+// red
+void    leds_error_on() {
+    GPIO_REG__OUTPUT    |=  0x01;
+}
+void    leds_error_off() {
+    GPIO_REG__OUTPUT    &= ~0x01;
+}
+void    leds_error_toggle() {
+    GPIO_REG__OUTPUT    ^=  0x01;
+}
+uint8_t leds_error_isOn() {
+    return (uint8_t)(GPIO_REG__OUTPUT & 0x01);
+}
+
+// orange
+void    leds_radio_on() {
+    GPIO_REG__OUTPUT    |=  0x02;
+}
+void    leds_radio_off() {
+    GPIO_REG__OUTPUT    &= ~0x02;
+}
+void    leds_radio_toggle() {
+    GPIO_REG__OUTPUT    ^=  0x02;
+}
+uint8_t leds_radio_isOn() {
+    return (uint8_t)(GPIO_REG__OUTPUT & 0x02)>>1;
+}
+
+// green
+void    leds_sync_on() {
+    GPIO_REG__OUTPUT    |=  0x04;
+}
+void    leds_sync_off() {
+    GPIO_REG__OUTPUT    &= ~0x04;
+}
+void    leds_sync_toggle() {
+    GPIO_REG__OUTPUT    ^=  0x04;
+}
+uint8_t leds_sync_isOn() {
+    return (uint8_t)(GPIO_REG__OUTPUT & 0x04)>>2;
+}
+
+// yellow

```

Comment:
these color labels seem wrong. please update with name of the LED on the silkscreen.